src/popover/popover.jsx

```diff
@@ -58,6 +60,7 @@ const Popover = React.createClass({
 
   getInitialState() {
     this.setPlacementThrottled = throttle(this.setPlacement, 100);
+    this.setPlacementScrolling = throttle(this.setPlacement.bind(this, true), 100);
```

Comment:
this broke scrolling. renaming `setPlacementScrolling ` to `setPlacementThrottledScrolled` fixes it.